modules/core/src/main/java/org/apache/fluo/core/log/TracingTransaction.java

```diff
@@ -96,6 +96,11 @@ private String encC(Map<Column, Bytes> ret) {
         + "=" + enc(e.getValue())));
   }
 
+  private String encMBC(Map<Bytes, Set<Column>> m) {
```

Comment:
I changed the method names.  I didn't think comments were needed since the methods are private, have descriptive name, and are single line implementations.